mmv1/products/bigquery/Dataset.yaml

```diff
@@ -26,7 +26,8 @@ docs:
     The API does accept both formats but it will always return the legacy format which results in Terraform
     showing permanent diff on each plan and apply operation.
 base_url: 'projects/{{project}}/datasets'
-self_link: 'projects/{{project}}/datasets/{{dataset_id}}'
+id_format: 'projects/{{project}}/datasets/{{dataset_id}}'
+self_link: 'projects/{{project}}/datasets/{{dataset_id}}?accessPolicyVersion=3'
 has_self_link: true
 delete_url: 'projects/{{project}}/datasets/{{dataset_id}}?deleteContents={{delete_contents_on_destroy}}'
```

Comment:
Should `accessPolicyVersion=3` be added to this `delete_url` ?

Reply:
I'll confirm this from the API team

Reply:
Looking at the API, `accessPolicyVersion` is only used for get/insert/patch/update requests